### ISSUE TITLE
[Concept] Hosts gossip

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#262]
At the moment my peers don't tell me about new hosts, I actively have to call their `getHosts`. We should think about a non-spammy host gossip protocol.

**Comment by sangaman** [comment]
I think gossiping known hosts could get messy, instead I'd prefer a manual way to send out `GET_HOSTS` packets or perhaps a configurable schedule of sending `GET_HOSTS` out periodically.

**Comment by kilrau** [comment]
Good idea, will revisit this later, first we'll fix https://github.com/ExchangeUnion/xud/issues/305
